.github/workflows/nightly-pipeline.yaml

```diff
@@ -76,6 +77,41 @@ jobs:
           propagate_failure: true
           trigger_workflow: true
           wait_workflow: true
+  rapids-cmake-build:
+    needs: get-run-info
+    runs-on: ubuntu-latest
+    steps:
+      - uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+          owner: rapidsai
+          repo: rapids-cmake
+          github_token: ${{ secrets.WORKFLOW_TOKEN }}
+          github_user: GPUtester
+          workflow_file_name: build.yaml
+          ref: ${{ fromJSON(needs.get-run-info.outputs.obj).branch }}
+          wait_interval: 120
+          client_payload: ${{ toJSON(fromJSON(needs.get-run-info.outputs.obj).payloads.rapids-cmake) }}
+          propagate_failure: true
+          trigger_workflow: true
+          wait_workflow: true
+  rapids-cmake-tests:
+    needs: [get-run-info, rapids-cmake-build]
```

Comment:
As it stands today, the `rapids-cmake-tests` do not depend on `rapids-cmake-build`. The build workflow produces packages of `rapids-core-dependencies` and the test workflow just runs rapids-cmake tests. These are independent (today, at least), but I'm okay with keeping the `needs:` entry here in case that ever changes -- it would look weird to deviate from the rest of RAPIDS, too ("why don't tests require builds...?").

Reply:
Agreed, I think it makes sense to keep this dependency for now.